reports/Arizona.md

```diff
@@ -49,7 +49,7 @@ On May 31, police forced protesters who were out after curfew into the Garfield
 
 Several other residents of the Garfield neighborhood have complained of police behavior that night, including one who says she passed out in her yard due to a tear-gas-induced asthma attack and awoke to find an officer handcuffing her.
 
-tags: arrest, tackle, tear-gas, bystander
+tags: arrest, tackle, tear-gas, bystander, zip-tie-likely
```

Comment:
we don't do "likely." The goal here is clear evidence so if you're not 100% certain, don't tag it

Reply:
I know, it was a placeholder while I looked for more info & waited on your input.

I did finally find some high-resolution images in [an MSN article](https://www.msn.com/en-us/news/crime/phoenix-crowds-protest-deaths-of-george-floyd-dion-johnson/ss-BB14SxJ4?parent-title=more-than-200-arrested-by-phoenix-police-during-protests-on-sunday-night&parent-ns=ar&parent-content-id=BB14Taoq&fullscreen=true#interstitial=3) showing zip ties were used in both downtown Phoenix and the Garfield neighborhood.

Here are a couple of the clearer ones from that MSN album:

<img src="https://user-images.githubusercontent.com/808746/92484997-66407c00-f1b8-11ea-842d-a6c1a1cef04b.png" width="300" height="200" />
<img src="https://user-images.githubusercontent.com/808746/92485016-69d40300-f1b8-11ea-8a81-62dfb30c92ba.png" width="300" height="200" />